packages/setup-component/src/core/constants.ts

```diff
@@ -1,5 +1,5 @@
 export const SETUP_COMPONENT_ID_SUFFIX = '/setup-component/'
-export const SETUP_COMPONENT_ID_REGEX = /\/setup-component\/(\d+).vue$/
+export const SETUP_COMPONENT_ID_REGEX = /\/setup-component\/(\d+).(vue|[jt]sx)$/
```

Comment:
Why change this? It's always `.vue`.
https://github.com/sxzz/unplugin-vue-macros/blob/d56c016dc7095e6f5b77ea596379f6d5def4bc6a/packages/setup-component/src/core/index.ts#L122-L124